.github/workflows/e2e-browser.yml

```diff
@@ -1,72 +1,68 @@
-name: End-to-end tests (browser)
+name: End-to-end browser tests
 
-on: 
-  deployment_status:
-    branches-ignore:
-      - 'dependabot/npm_and_yarn/**'
-  workflow_dispatch:
+on: [push, workflow_dispatch]
 
 env:
   CI: true
 jobs:
-  test:
-    runs-on: ubuntu-20.04
-    if: ${{github.event.deployment_status.state == 'success'}}
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # Available OS's: https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners
+        os: [ubuntu-20.04]
+        node-version: [14.x]
     steps:
       - uses: actions/checkout@v2.3.4
-      - name: Use Node.js 12.x
+      - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v2.1.5
+        with:
+          node-version: ${{ matrix.node-version }}
       - name: Cache node modules
         uses: actions/cache@v2.1.5
         env:
           cache-name: cache-node-modules
         with:
-          path: e2e/browser/node_modules
+          path: node_modules
```

Comment:
Heh, good one. I wonder if maybe the pipelines could be sped up some more by also caching the `/packages/*/node_modules/`?

Reply:
That's an excellent question. I'll give it a try and see, the action documentation seems to lean towards this.